scripts/srt-dev.lua

```diff
@@ -0,0 +1,887 @@
+}
+fields.resv = ProtoField.uint8("srt_dev.resv", "Resv", base.DEC, state_table, 0xFC)
+fields.ext_KK_state = ProtoField.uint8("srt_dev.ext_KK_state", "KK_state", base.HEX, KK_state_select, 0x3)
+fields.KEKI = ProtoField.uint32("srt_dev.KEKI", "KEKI", base.DEC)
+fields.cipher = ProtoField.uint8("srt_dev.cipher", "Cipher", base.DEC)
+fields.auth = ProtoField.uint8("srt_dev.auth", "auth", base.DEC)
+fields.SE = ProtoField.uint8("srt_dev.SE", "SE", base.DEC)
+fields.resv1 = ProtoField.uint8("srt_dev.resv1", "resv1", base.DEC)
+fields.resv2 = ProtoField.uint16("srt_dev.resv2", "resv2", base.DEC)
+fields.slen = ProtoField.uint8("srt_dev.slen", "Salt length(bytes)/4", base.DEC)
+fields.klen = ProtoField.uint8("srt_dev.klen", "SEK length(bytes)/4", base.DEC)
+fields.salt = ProtoField.uint32("srt_dev.salt", "Salt key", base.DEC)
+fields.wrap = ProtoField.none("srt_dev.wrap", "Wrap key(s)", base.NONE)
+
+-- Handshake packet, ext_type == SRT_CMD_SMOOTHER filed
+fields.smoother_block = ProtoField.none("srt_dev.smoother_block", "Smoother Block", base.NONE)
+
+-- Handshake packet, ext_type == SRT_CMD_SID filed
+fields.sid_block = ProtoField.none("srt_dev.sid_block", "SID Block", base.NONE)
+
+-- SRT flags
+local flag_state_select = {
+	[0] = "Not set",
+	[1] = "Set"
+}
+fields.srt_opt_tsbpdsnd = ProtoField.uint32("srt_dev.srt_opt_tsbpdsnd", "SRT_OPT_TSBPDSND", base.HEX, flag_state_select, 0x1)
+fields.srt_opt_tsbpdrcv = ProtoField.uint32("srt_dev.srt_opt_tsbpdrcv", "SRT_OPT_TSBPDRCV", base.HEX, flag_state_select, 0x2)
+fields.srt_opt_haicrypt = ProtoField.uint32("srt_dev.srt_opt_haicrypt", "SRT_OPT_HAICRYPT", base.HEX, flag_state_select, 0x4)
+fields.srt_opt_tlpktdrop = ProtoField.uint32("srt_dev.srt_opt_tlpktdrop", "SRT_OPT_TLPKTDROP", base.HEX, flag_state_select, 0x8)
+fields.srt_opt_nakreport = ProtoField.uint32("srt_dev.srt_opt_nakreport", "SRT_OPT_NAKREPORT", base.HEX, flag_state_select, 0x10)
+fields.srt_opt_rexmitflg = ProtoField.uint32("srt_dev.srt_opt_rexmitflg", "SRT_OPT_REXMITFLG", base.HEX, flag_state_select, 0x20)
+fields.srt_opt_stream = ProtoField.uint32("srt_dev.srt_opt_stream", "SRT_OPT_STREAM", base.HEX, flag_state_select, 0x40)
+
+-- ACK fileds
+fields.last_ack_pack = ProtoField.uint32("srt_dev.last_ack_pack", "Last ACK Packet Sequence Number", base.DEC)
+fields.rtt = ProtoField.uint32("srt_dev.rtt", "Round Trip Time", base.DEC)
+fields.rtt_variance = ProtoField.uint32("srt_dev.rtt_variance", "Round Trip Time Variance", base.DEC)
+fields.buf_size = ProtoField.uint32("srt_dev.buf_size", "Available Buffer Size", base.DEC)
+fields.pack_rcv_rate = ProtoField.uint32("srt_dev.pack_rcv_rate", "Packet Receiving Rate", base.DEC)
+fields.est_link_capacity = ProtoField.uint32("srt_dev.est_link_capacity", "Estimated Link Capacity", base.DEC)
+fields.rcv_rate = ProtoField.uint32("srt_dev.rcv_rate", "Receiving Rate", base.DEC)
+
+-- ACKACK fileds
+fields.ack_seq_num = ProtoField.uint32("srt_dev.ack_seq_num", "ACK sequence number", base.DEC)
```

Comment:
ACKACK messages definitely don't contain ACK sequence number.

Both ACK and ACKACK messages contain the "ACK journal" number, which is a number that starts from 1 and increases with every ACK sent. ACK message contains this number so that the returning ACKACK message can find the original ACK message for which this ACKACK is a response. This is a normal technique used in request-response cycles to identify the response for which request it was sent.

I'd really recommend to look at the SRT dissector in the wireshark sources and follow its footsteps.